.github/workflows/cypress.yml

```diff
@@ -45,19 +45,28 @@ jobs:
         key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
     - name: Install dependencies
       run: npm install
-    - name: Set the core version and plugins config 
+    - name: Set the core version and plugins config
       run: ./tests/bin/set-wp-config.js --core=${{ matrix.core.version }} --plugins=./${{ github.event.repository.name }}
     - name: Set up WP environment
       run: npm run env:start
     - name: Test
       run: npm run cypress:run
-    - name: Upload artifacts
-      uses: actions/upload-artifact@v2
+    - name: Update summary
+      if: always()
```

Comment:
I added this in 065f65801dbad8f476d066b5a0f990da6aa2b22d so the summary displays in the GitHub actions whether the tests pass or fail.